docs/understanding-radicle/glossary.md

```diff
@@ -77,13 +77,13 @@ A device running the Radicle Link protocol.
 ## Device ID
 The encoding of a peer's public key tied to their device. In the Beta, there will
 only be one Device ID per user (See
-[Architecture](how-it-works.md/#architecture))
+[Architecture][hiw-ar])
```

Comment:
This link/heading does not exist. Is it supposed to link to [Key Architecture](http://localhost:3000/docs/understanding-radicle/how-it-works/#key-architecture)?

Reply:
Probably.

Reply:
No, we're deleting this section. Will have to update it